lib/iris/tests/unit/cube/test_Cube__operators.py

```diff
@@ -160,5 +161,26 @@ def test_div_cubes(self):
         self.assertTrue(isinstance(data, np.ndarray))
         self.assertEqual(data.shape, ())
 
+
+class Test_Masked_Lazy_Maths(tests.IrisTest):
+
+    def build_lazy_cube(self):
```

Comment:
We have so many examples of this sort of thing in Iris tests! I'm not suggesting you do it here, but a lazy cube provider in `iris.tests.stock` might be a good thing...